src/main/scala/com/gu/scanamo/package.scala

```diff
@@ -4,7 +4,7 @@ import com.gu.scanamo.query._
 
 package object scanamo {
 
-  object syntax {
+  object syntax extends Scannable.ToScannableOps with Queryable.ToQueryableOps {
```

Comment:
Why use traits and inheritance here instead of importing the operations from objects?